forum/qa-include/pages/message.php

```diff
@@ -68,8 +68,13 @@
 
 //	Check the user exists and work out what can and can't be set (if not using single sign-on)
 
-	if ( !qa_opt('allow_private_messages') || !is_array($toaccount) || ($toaccount['flags'] & QA_USER_FLAGS_NO_MESSAGES) )
-		return include QA_INCLUDE_DIR.'qa-page-not-found.php';
+        if (!qa_opt('allow_private_messages') || !is_array($toaccount) || ($toaccount['flags'] & QA_USER_FLAGS_NO_MESSAGES)) {

```

Comment:
Jak miałbym być szczery to qa_opt() bym przeniósł do zmiennej i w ifie wykonałbym wywołanie nazwy tej zmiennej.